db/models.py

```diff
@@ -50,3 +54,66 @@ class MovieSession(models.Model):
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(settings.AUTH_USER_MODEL,
+                             on_delete=models.CASCADE,
+                             related_name="user")
```

Comment:
Related name is not correct, replace `user` with `orders`. The related name should be in the plural form and represent how to refer to this model from the one mentioned in the `ForeignKey`.